deploy/single-v2/all-in-one-dbless.yaml

```diff
@@ -6,11 +6,10 @@ metadata:
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    controller-gen.kubebuilder.io/version: v0.6.2
-  creationTimestamp: null
   name: kongclusterplugins.configuration.konghq.com
 spec:
+  conversion:
```

Comment:
The definition of conversion is to provide backwards compatibility, and it does not modify validation schema, we do not really need it.

Reply:
Sounds good, removed in 0da2331b7282d9a15c28817fa7dfc5aea8f2a951

Reply:
Yep, the lint CI was catching this: I'm re-running the generators :+1: